### ISSUE TITLE
Detailed stats

### ISSUE THREAD

**Issue by jdtanner** [orf/gping#67]
Hi detailed stats are shown in the animated gif but there doesn't appear to be instructions as to how to enable them?

JT

**Comment by joshk0** [comment]
It looks like the detailed stats are available in the tip of master but not in the released snap or pip version. So, you can do this:

```
pip3 uninstall pinggraph
git clone https://github.com/orf/gping.git
cd gping
pip3 install -r requirements.txt
python setup.py install
```

If you can confirm this works for you, lets retitle this issue as 'make a new pypi / snap release'?

**Comment by orf** [comment]
Hey, I have released version 1.2 of gping. You can install it with `pip install --upgrade pinggraph`.

I'm working on re-writing this tool in Rust (under the [rust branch](https://github.com/orf/gping/tree/rust)), hence the slow replies to the issues and PRs here. I'll close this, let me know if there are any issues.
